test/src/test/java/org/corfudb/runtime/clients/NettyCommTest.java

```diff
@@ -321,6 +330,81 @@ public void nettySaslWrongPassword() throws Exception {
             });
     }
 
+    @Test
+    public void testTlsUpdateServerTrust() throws Exception {
+        reloadedTrustManagerTestHelper(false);
+    }
+
+    @Test
+    public void testTlsUpdateClientTrust() throws Exception {
+        reloadedTrustManagerTestHelper(true);
+    }
+
+    private void reloadedTrustManagerTestHelper(boolean replaceClientTrust) throws Exception {
```

Comment:
Can you add comments for this method/test

Reply:
Fixed.